addons/phone_validation/tools/phone_validation.py

```diff
@@ -14,6 +15,22 @@
     import phonenumbers
 
     def phone_parse(number, country_code):
+        # MONKEY PATCHING for the phone number of Ivory Coast and with the lib at least phonenumbers==8.9.10 it's working well with phonenumbers==8.12.32
+        if parse_version(phonenumbers.__version__) < parse_version('8.12.32') and country_code == 'CI':
```

Comment:
In order to determine whether the library is already monkey-patched, you could also check whether `phonenumbers.data.region_CI.PHONE_METADATA_CI.general_desc.possible_length[0] == 10`

Reply:
That's a demonstration that we should not monkey patch. It is a deployment issue. If you don't update your librairies, your code is limited. That's not Odoo's job to try to re-develop features already done in librairies... if you stay with Odoo 12 you don't have the features done in v15. It seems logical to me.